json/products/mozilla-concrete-mortar.json

```diff
@@ -1,8 +1,11 @@
 {
   "description": {
-    "en": "Some mortar"
+    "en": "Mortar"
   }, 
   "img": "https://raw.githubusercontent.com/mozilla/payments-config/master/payments_config/assets/mortar.png", 
+  "price": {
```

Comment:
i don't think we can get rid of amount, we need something that has  a decimal value, i'm using it in multiple places